pkgs/applications/misc/xdragon/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3 }:
```

pkgconfig is an alias
```suggestion
{ stdenv, fetchFromGitHub, pkg-config, gtk3 }:
```